pkg/common/path.go

```diff
@@ -14,30 +14,43 @@ import (
 	"sigs.k8s.io/kustomize/kyaml/kio/kioutil"
 )
 
+const stdinDash = "-"
```

Comment:
I'm not sure we should use this convention. I know we do it with kubectl, but I think we can just assume that no arguments means stdin.

Reply:
Right now, both "no arguments" and "dash" mean StdIn. If you think it  should only be "no arguments", please let me know.

Reply:
I think I slightly prefer that we don't support the dash argument. It is easier to add it later than to remove it once it is in.

Reply:
Removed dash as an argument; only "no argument" means stdin now.